README.md

```diff
@@ -81,30 +81,30 @@ You can call any method on authenticated user. For example, let's send message t
   var result = await client.GetContactsAsync();
 
   //find recipient in contacts
-  var user = result.Users.lists
+  var user = result.Users
 	  .Where(x => x.GetType() == typeof (TLUser))
 	  .Cast<TLUser>()
-	  .FirstOrDefault(x => x.phone == "<recipient_phone>");
+	  .FirstOrDefault(x => x.Phone == "<recipient_phone>");
 	
   //send message
-  await client.SendMessageAsync(new TLInputPeerUser() {user_id = user.id}, "OUR_MESSAGE");
+  await client.SendMessageAsync(new TLInputPeerUser() {UserId = user.Id}, "OUR_MESSAGE");
 ```
 
 Full code you can see at [SendMessage test](https://github.com/sochix/TLSharp/blob/master/TLSharp.Tests/TLSharpTests.cs#L87)
 
 To send message to channel you could use the following code:
 ```csharp
   //get user dialogs
-  var dialogs = await client.GetUserDialogsAsync();
+  var dialogs = (TLDialogs) await client.GetUserDialogsAsync();
 
   //find channel by title
-  var chat = dialogs.chats.lists
+  var chat = dialogs.Chats
```

Comment:
same here about missing .Lists

Reply:
and same reason here, there is no need for it and having it cause compile error

Reply:
ok